gini/gini-vision-lib-ios
# Improve multipage memory usage

### Description
After taking pictures from both the camera and pickers, they weren't downsized to their proper size in the multipage collection views, increasing the memory usage unnecessarily.
All the set-up logic for the cells has been moved to a new collection presenter, which take over this task now.

### How to test
Run the example app without in the `develop` branch, import 10 pictures and check the memory usage. Do the same in the current branch and compare the results

### Merging
Automatic
